### PR TITLE
Making Arp clear conditional for T2 in qos_sai_test

### PR DESCRIPTION
**Pull request by ansrajpu-git**
### Description of PR

This PR is impacted by the PR # https://github.com/sonic-net/sonic-mgmt/pull/7819 

For T2 topology the arp table is not getting refreshed/updated with the dynamic Arp entries after arp clear on the DUT.

The Apr populate test broadcast the arp packet and sleep for 8 seconds. Within this time frame no arp is learnt on the DUT side & fails at qos test run with below error as the port_ip's are missing in Arp table:

"Traceback (most recent call last):", "  File \"saitests/py3/sai_qos_tests.py\", line 3934, in runTest", "    self, 0, src_port_id, pkt_dst_mac, dst_port_ip, src_port_ip, src_port_vlan", "  File \"saitests/py3/sai_qos_tests.py\", line 198, in get_rx_port", "    result.port, result.format()))", "AssertionError: Expected packet was not received. Received on port:None ========== EXPECTED =========="

Summary:
For  T2-topology, in ARPpopulate  test, found below issues -
1.After arp clear, even if the sleep time is increased to 30 secs on the Arppopulate test the Arp is not learnt. We observed that the arp packets sent in test are dropped on the DUT during the run.
2.In the test, since there is no return value, the arp entries left unverified on the DUT before the qos test run.

Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [x] 202205

### Approach
Since the PR# https://github.com/sonic-net/sonic-mgmt/pull/7819 resolve the dualtor's issue
The step to clear arp & fdb are made conditional for topo type as dualtor only

#### What is the motivation for this PR?
Since all the qos test were impacted by the ArpPopulate change, all test cases failed with error -

"Traceback (most recent call last):", "  File \"saitests/py3/sai_qos_tests.py\", line 3934, in runTest", "    self, 0, src_port_id, pkt_dst_mac, dst_port_ip, src_port_ip, src_port_vlan", "  File \"saitests/py3/sai_qos_tests.py\", line 198, in get_rx_port", "    result.port, result.format()))", "AssertionError: Expected packet was not received. Received on port:None ========== EXPECTED ==========" 

#### How did you do it?
The step to clear arp & fdb are made conditional for topo type as dualtor only.

#### How did you verify/test it?
Re-run the  qos  test suite

### CODE REVIEW

File: tests/qos/qos_sai_base.py [hunk 1]
```diff
@@ -1291,8 +1291,10 @@ def populateArpEntries(
 
         dut_asic = duthost.asic_instance(enum_frontend_asic_index)
 
-        dut_asic.command('sonic-clear fdb all')
-        dut_asic.command('sonic-clear arp')
+        if 'dualtor' in tbinfo['topo']['name']:
```
Comment (vmittal-msft) [inline tests/qos/qos_sai_base.py:1294]: Can we make it conditional for T2 only since we are seeing issue on T2 chassis ?

Comment (ansrajpu-git) [reply tests/qos/qos_sai_base.py]: thanks for the comment. Incorporated it ,please check

**Comment by ansrajpu-git** [comment]
This PR can be ignored as the changes are taken care in PR - https://github.com/sonic-net/sonic-mgmt/pull/8377 raised in 202205 branch

**Comment by vmittal-msft** [comment]
This fix is not relevant anymore. Closing.

**Pull request closed without merging**